app/views/index.njk

```diff
@@ -1,89 +1,98 @@
+{% set pageTitle = "Title needed" %}
+{% set pageDescription = "This is the lead description needed" %}
+
 {% extends 'includes/layout.njk' %}
 
 {% block outerBody %}
-<section class="nhsuk-hero">
-  <div class="nhsuk-width-container nhsuk-hero--border">
-    <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-two-thirds">
-        <div class="nhsuk-hero__wrapper">
-          <h1 class="nhsuk-u-margin-bottom-3">Technical Review Group and Governance</h1>
-          <p class="nhsuk-body-l nhsuk-u-margin-bottom-0">Helping technical architects and project professionals to navigate our technical governance process.</p>
-        </div>
-      </div>
-    </div>
+  <nav class="nhsuk-breadcrumb{% if params.classes %} {{ params.classes }}{% endif %}" aria-label="Breadcrumb"
```

Comment:
The homepage probably shouldn't have a Breadcrumb as it serves no purpose in navigation and only links to itself